templates/partials/contents.html

```diff
@@ -0,0 +1,15 @@
+{% for item in content_block.contents %}
```

Comment:
This is a little confusing as contents aren't always in a block content element. It would be clearer if a `contents` variable was set in templates that include this template. So using `{% set contents = content_block.contents %}` in the parent template and this would just be `{% for item in contents %}`.